src/snapshotVersion.js

```diff
@@ -3,9 +3,19 @@ const createVersion = require('./lib/createVersion.js');
 
 module.exports = (argsParser = process.argv) => {
     program
-        .command('create <version> [siteDir]')
-        .action((version, siteDir = ".") => {
-            createVersion.create(version, siteDir);
+        .command('create')
+        .storeOptionsAsProperties()
+        .requiredOption('--version <type>', 'The value of the new version to create.')
+        .option('--siteDir <type>', 'The absolute or relative path to the `website` directory of the Docusaurus V1 project.', '.')
+        .option('--assetType <type...>', 'The file type of the static file type to be versioned.')
```

Comment:
Let's change the option name to `--staticAssetDir` (unless you have a better name). The description also needs to be changed. The "file type" is mis-leading. It could be interpreted as ".img", ".html", ".css", etc. But, that's not what this parameter is expecting or going to handle. This parameter will version an entire directory.